### PR TITLE
python-stratify recipe

### PR DESCRIPTION
**Pull request by bjlittle**

### CODE REVIEW

**Comment by bjlittle** [comment]
Having issues with the windows build, see https://github.com/conda-forge/staged-recipes/pull/2393

Happy to skip the windows build just now, until this is resolved.

**Comment by bjlittle** [comment]
Ping @lbdreyer

File: python-stratify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Vectorized interpolators that are especially useful for Nd vertical interpolation/stratification of atmospheric and oceanographic datasets' 
+
+extra:
+  recipe-maintainers:
+    - bjlittle
+    - lbdreyer
+    - ocefpaf
+    - pelson
```
Comment (lbdreyer) [inline python-stratify/meta.yaml]: I think the recipe maintainers is only important for conda forge recipes

**Comment by bjlittle** [comment]
@lbdreyer Let's wait to see if the `conda-forge` guys can help out with a fix for the windows build failure, rather than merge this as is with the windows build skipper ...

File: python-stratify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/SciTools-incubator/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (ocefpaf) [inline python-stratify/meta.yaml]: You can skip `[win and py27]` only.

**Comment by bjlittle** [comment]
Ping @lbdreyer

**Comment by lbdreyer** [comment]
So the latest release of python-stratify can build on windows correctly.

@bjlittle I'm gonna squash this into a single commit.

**Pull request merged**